pyproject.toml

```diff
@@ -13,13 +13,13 @@ authors = [
     { name="Sam Webster", email="13457618+samwebster@users.noreply.github.com" },
     { name="Tom Minka", email="8955276+tminka@users.noreply.github.com" },
     { name="Siddharth Krishna", email="siddharth-krishna@users.noreply.github.com" },
-    { name="Olexandr Balyk", email="olex@energymodelling.club" },
+    { name="Olexandr Balyk", email="olex@energymodelling.club" }
```

Comment:
Curious why we're removing trailing commas from this file? Usually it's a good practice to have them, if the language allows, because then git diffs are smaller when adding a new item to the end of a list.

Reply:
Tbh, I don't remember why I removed them there 7 months ago and whether it had any effect... Do you think it is worth undoing it?

Reply:
Ah haha didn't notice this was from a long time ago. Well, a bit, because at least the squash merged commit of this PR will be smaller and if we add e.g. a classifier later it won't create a bigger diff? But I can do it if you don't mind.

Reply:
Sure, just go ahead!